config/development_polygon.toml

```diff
@@ -0,0 +1,216 @@
+soft_limit = 10
+tier = 2
+
+[balanced_rpcs.maticvigil]
+disabled = false
+display_name = "Maticvigil"
+http_url = "https://rpc-mainnet.maticvigil.com"
+soft_limit = 10
+tier = 2
+
+[balanced_rpcs.matic-network]
+disabled = false
+display_name = "Matic Network"
+http_url = "https://rpc-mainnet.matic.network"
+soft_limit = 10
+tier = 1
+
+[balanced_rpcs.chainstack]
+disabled = false
+display_name = "Chainstack"
+http_url = "https://matic-mainnet.chainstacklabs.com"
+soft_limit = 10
+tier = 2
+
+[balanced_rpcs.bware]
+disabled = false
+display_name = "Bware Labs"
+http_url = "https://matic-mainnet-full-rpc.bwarelabs.com"
+soft_limit = 10
+tier = 2
+
+[balanced_rpcs.bware_archive]
+disabled = false
+display_name = "Bware Labs Archive"
+http_url = "https://matic-mainnet-archive-rpc.bwarelabs.com"
+soft_limit = 10
+tier = 2
+
+[balanced_rpcs.polygonapi]
+disabled = false
+display_name = "Polygon API"
+http_url = "https://polygonapi.terminet.io/rpc"
+soft_limit = 10
+tier = 2
+
+[balanced_rpcs.one-rpc]
+disabled = false
+display_name = "1RPC"
+http_url = "https://1rpc.io/matic"
+soft_limit = 10
+tier = 2
+
+[balanced_rpcs.fastrpc]
+disabled = false
+display_name = "1RPC"
+http_url = "https://polygon-mainnet.rpcfast.com?api_key=xbhWBI1Wkguk8SNMu1bvvLurPGLXmgwYeC4S6g2H7WdwFigZSmPWVZRxrskEQwIf"
+soft_limit = 10
+tier = 2
+
+[balanced_rpcs.unifra]
+disabled = false
+display_name = "Unifra"
+http_url = "https://polygon-mainnet-public.unifra.io"
+soft_limit = 10
+tier = 2
+
+[balanced_rpcs.onfinality]
+disabled = false
+display_name = "Onfinality"
+http_url = "https://polygon.api.onfinality.io/public"
+soft_limit = 10
+tier = 2
+
+[balanced_rpcs.alchemy]
+disabled = false
+display_name = "Alchemy"
+heept_url = "https://polygon-mainnet.g.alchemy.com/v2/demo"
+soft_limit = 10
+tier = 2
+
+[balanced_rpcs.blockpi]
+disabled = false
+display_name = "Blockpi"
+http_url = "https://polygon.blockpi.network/v1/rpc/public"
+soft_limit = 100
+tier = 2
+
+[balanced_rpcs.polygon]
+backup = true
+disabled = false
+display_name = "Polygon"
+http_url = "https://polygon-rpc.com"
+soft_limit = 10
+tier = 2
+
+[balanced_rpcs.pokt]
+disabled = false
+display_name = "Pokt"
+http_url = "https://poly-rpc.gateway.pokt.network"
+soft_limit = 10
+tier = 2
+
+[balanced_rpcs.ankr]
+backup = true
+disabled = false
+display_name = "Ankr"
+http_url = "https://rpc.ankr.com/polygon"
+soft_limit = 10
+tier = 2
+
+[balanced_rpcs.blastapi]
+backup = true
+disabled = true
+display_name = "Blast"
+http_url = "https://polygon-mainnet.public.blastapi.io"
+hard_limit = 10
+soft_limit = 10
+tier = 2
+
+[balanced_rpcs.omnia]
+disabled = true
+display_name = "Omnia"
+http_url = "https://endpoints.omniatech.io/v1/matic/mainnet/public"
+soft_limit = 10
+tier = 2
+
+[balanced_rpcs.bor]
+disabled = true
+display_name = "bor"
+http_url = "https://polygon-bor.publicnode.com"
+soft_limit = 10
+tier = 2
+
+[balanced_rpcs.blxr]
+disabled = false
+display_name = "blxr"
```

Comment:
doesn't really matter, but having the "display_name" is only useful if its different then the name in the key (both here are blxr)

Reply:
ah ok good to know. i kept the names that i capitalized and removed the ones that have case-matching character-sequences